site/config-overrides.md

```diff
@@ -175,59 +177,81 @@ The secret macro `SECRETNAME` field must reference the name of a Kubernetes secr
+  * E.g. use `d:` to reference config.xml beans and attributes, `f:` for `add` and `replace` `domain-fragment` verbs, and `s:` to reference the situational config schema.
+* It is a best practice to use XML namespace abbreviations `jms:`, `jdbc:`, and `wldf:` respectively for JMS, JDBC, and WLDF module override files.
+
+## Override template samples
 
-**IMPORTANT: Your overrides may not take effect if you use 'add' for an MBean attribute that already has a value specified in your original configuration, or use 'replace' for an attribute that does not already exist.**
+Here are some sample template override files. 
 
 ### Overriding `config.xml`
 
-The following `config.xml` override file demonstrates setting the `max-message-size` field on a WebLogic Server named `admin-server`, and replacing the `public-address` and `public-port` fields with values obtained from a secret named `test-host` with keys `hostname` and `port`.
+The following `config.xml` override file demonstrates:
+
+ *  Setting the `max-message-size` field on a WebLogic Server named `admin-server`.  It assumes the original `config.xml` does not define this value, and so uses `add` instead of `replace`.
+ *  Sets the `public-address` and `public-port` fields with values obtained from a secret named `test-host` with keys `hostname` and `port`. It assumes the original config.xml already sets these fields, and so uses `replace` instead of `add`.
+ *  Sets two debug settings. It assumes the original config.xml does not have a `server-debug` stanza, so it uses `add` throughout the entire stanza.
 
 ```
 <?xml version='1.0' encoding='UTF-8'?>
-<domain xmlns="http://xmlns.oracle.com/weblogic/domain"
-        xmlns:f="http://xmlns.oracle.com/weblogic/domain-fragment"
-        xmlns:s="http://xmlns.oracle.com/weblogic/situational-config">
-  <name>${env:DOMAIN_NAME}</name>
-  <server>
-    <name>admin-server</name>
-    <max-message-size f:combine-mode="add">78787878</max-message-size>
-    <network-access-point>
-      <name>T3Channel</name>
-      <public-address f:combine-mode="replace">${secret:test-host.hostname}</public-address>
-      <public-port f:combine-mode="replace">${secret:test-host.port}</public-port>
-    </network-access-point>
-  </server>
-</domain>
+<d:domain xmlns:d="http://xmlns.oracle.com/weblogic/domain"
+          xmlns:f="http://xmlns.oracle.com/weblogic/domain-fragment"
+          xmlns:s="http://xmlns.oracle.com/weblogic/situational-config" >
+    <d:name>${env:DOMAIN_NAME}</d:name>
+    <d:server>
+        <d:name>admin-server</d:name>
+        <d:max-message-size f:combine-mode="add">78787878</d:max-message-size>
+        <d:server-debug f:combine-mode="add">
+          <d:debug-server-life-cycle f:combine-mode="add">false</d:debug-server-life-cycle>
+          <d:debug-jmx-core f:combine-mode="replace">false</d:debug-jmx-core>
```

Comment:
"debug-jmx-core" should be "add" instead of replace since the server-debug is "add"